src/components/__tests__/AddRoom.test.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import AddRoom from '../Rooms/AddRoom';
+import store from '../../redux/configureStore';
+
+describe('AddRoom', () => {
+  render(
+    <Provider store={store}>
+      <AddRoom />
+    </Provider>,
+  );
+
+  test('Test the dom for success rendering elements', () => {
+    const title = screen.getByPlaceholderText('Description');
+    expect(title).toBeInTheDocument();
+  });
```

Comment:
- [ ] For some reason, this particular test is not passing on my end:

![image](https://user-images.githubusercontent.com/57421823/195331311-5d9fdabe-4073-4271-983c-1f490a7bd621.png)

Kindly check if this test is passing on your env, and if it is not, please kindly fix it to ensure your app's integrity 💪

Reply:
Thank you so much for the feedback, Miguel. I will handle this issue.